tests/LowestCommonAncestorTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Relaxed\LCA\Test;
+
+use Fhaculty\Graph\Graph;
+use Relaxed\LCA\LowestCommonAncestor;
+use Fhaculty\Graph\Vertex;
+use Graphp\Algorithms\Search\BreadthFirst;
+
+class LowestCommonAncestorTest extends \PHPUnit_Framework_TestCase
+{
+    public $graph;
+    public function testlca()
+    {
+        $graph = new Graph();
+
+        $one = $graph->createVertex('One');
```

Comment:
This part should be replaced with one method. For example `$vertices = generateVertices($graph, 20);` - this will generate an array with 20 vertices. 
Example of a method that will generate vertices:
```
public function generateVertices(Graph $graph, $count = 5)
    {
        $vertices = [];
        for ($i = 1; $i <= $count; $i++)
        {
            $node_id = "node_$i";
            $vertices[$node_id] = $graph->createVertex($node_id);
        }
        return $vertices;
    }
```

When you have the array with vertices create the edges: 
```
$vertices['node_1']->createEdgeTo($vertices['node_3']);
$vertices['node_1']->createEdgeTo($vertices['node_2']);
...
```